src/vm/eventtrace.cpp

```diff
@@ -4875,7 +4900,7 @@ VOID ETW::InfoLog::RuntimeInformation(INT32 type)
 
             LPCGUID comGUID=&g_EEComObjectGuid;
 
-            LPWSTR lpwszCommandLine = W("");
+            LPWSTR lpwszCommandLine = (LPWSTR)W("");
             LPWSTR lpwszRuntimeDllPath = (LPWSTR)dllPath;
```

Comment:
Please don't cast away constness and rather make the variable LPCWSTR (and change the signature of the functions that get that as parameter to accept LPCWSTR).